core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```diff
@@ -110,6 +114,17 @@ <h3 class="oppia-mobile-question-header" title="{{questionHeader}}">
 </div>
 
 <style>
+  .modal-header {
```

Comment:
Can you confirm (by posting a video) that other modals on /contributor-dashboard which also use .modal-header are not affected by the style changes here?